packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx

```diff
@@ -163,7 +169,21 @@ export class CommandBar extends BaseComponent<ICommandBarProps, ICommandBarState
     this.refs.focusZone.focus();
   }
 
-  private _renderItemInCommandBar(item: ICommandBarItemProps, index: number, expandedMenuItemKey: string, isFarItem?: boolean) {
+  private _renderCommandBarItem(item: ICommandBarItemProps, index: number, expandedMenuItemKey: string, isFarItem?: boolean) {
+    switch (item.itemType) {
+      case ContextualMenuItemType.Divider:
+        return this._renderSeparator(item, index, isFarItem);
+      case ContextualMenuItemType.Header:
+      case ContextualMenuItemType.Section:
+        // Unsupported
+        return null;
+      case ContextualMenuItemType.Normal:
+      default:
+        return this._renderNormalItem(item, index, expandedMenuItemKey);
```

Comment:
This should instead be
```ts
      case ContextualMenuItemType.Normal:
        return this._renderNormalItem(item, index, expandedMenuItemKey);
      default:
        // If we ever add a new ContextualMenuItemType in the future without adding to this switch,
        // it will result in a build break
        assertNever(item.itemType);
```

Reply:
Null case should go to normal though, correct?

Reply:
I added an undefined check above the switch so users won't be forced to add itemType to all of their commandbar items.